runtime/kusama/src/lib.rs

```diff
@@ -1559,7 +1605,32 @@ pub mod migrations {
 				StorageVersion::new(1).put::<Historical>();
 			}
 
-			RocksDbWeight::get().reads_writes(2, 2)
+			let storage_version = Democracy::on_chain_storage_version();
+			if storage_version < 1 {
+				StorageVersion::new(1).put::<Democracy>();
+			}
+
+			let storage_version = Council::on_chain_storage_version();
+			if storage_version < 4 {
```

Comment:
Why is this needed?  
I thought the pallet storage is still there.

Reply:
I swore that the on-chain versions had not been set causing the on-runtime-upgrade to fail when I added these, but I can't reproduce it now.. Probably a mistake, I'll remove them.